deepchecks/tabular/metric_utils/scorers.py

```diff
@@ -293,7 +293,18 @@ def _run_score(self, model, dataset: 'tabular.Dataset'):
                 f'{doclink_str}')
         if self.possible_classes is not None:
             updated_model, transformed_label_col = self._wrap_classification_model(model, dataset.label_col)
-            return self.scorer(updated_model, dataset.features_columns, transformed_label_col)
+            scores = self.scorer(updated_model, dataset.features_columns, transformed_label_col)
+            # Some metrics like accuracy per class returns score only for classes exists in the data. We want to have
+            # score for each possible class
+            if isinstance(scores, t.Sized) and len(scores) < len(self.possible_classes):
+                assert len(scores) == len(dataset.classes_in_label_col)
+                class_scores = list(zip(dataset.classes_in_label_col, scores))
+                missing_classes = set(self.possible_classes) - set(dataset.classes_in_label_col)
+                class_scores.extend([(clas, np.nan) for clas in missing_classes])
```

Comment:
In other places we used pd.NA see tabular SingleDatasetPerformance line 74
not sure which is better but make them the same

Reply:
it's different, the pd.NA there represents class, in here it represents a score. I must return here numpy array so it must be np.nan. 
I can change there from pd.NA to np.nan but then it thinks it's a numeric column so I also have to do `str(class)`. so anyway I think it's better as it is now